File: cpp/cada_linha.cpp
#include<iostream>

using namespace std;

int main(){
    int n, maior;

    cout << "Qual a ordem da matriz? ";
    cin >> n;

    int matriz[n][n], maiorlinha[n];

    for(int i=0; i<n; i++){
        for(int j=0; j<n; j++){
            cout << "Elemento [" << i << "," << j << "]: ";
            cin >> matriz[i][j];
        }
    }

    for(int i=0; i<n; i++){
        maior = matriz[i][0];
        for(int j=1; j<n; j++){
            if(maior < matriz[i][j]){
                maior = matriz[i][j];
            }
        }
        maiorlinha[i] = maior;
    }

    cout << "MAIOR ELEMENTO DE CADA LINHA:" << endl;

    for(int i=0; i<n; i++){
        cout << maiorlinha[i] << endl;
    }

    return 0;
}

File: java/cada_linha.java
import java.util.Locale;
import java.util.Scanner;

public class cada_linha {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n, maior;

	    System.out.print("Qual a ordem da matriz? ");
	    n = sc.nextInt();
	
	    int[][] matriz = new int[n][n];
	    int[] maiorlinha = new int[n];
	
	    for (int i=0; i<n; i++) {
	        for (int j=0; j<n; j++) {
				System.out.printf("Elemento [%d,%d]: ", i, j);
	            matriz[i][j] = sc.nextInt();
	        }
	    }
	
	    for (int i=0; i<n; i++) {
	        maior = matriz[i][0];
	        for (int j=1; j<n; j++) {
	            if (maior < matriz[i][j]) {
	                maior = matriz[i][j];
	            }
	        }
	        maiorlinha[i] = maior;
	    }
	
	    System.out.println("MAIOR ELEMENTO DE CADA LINHA:");
	
	    for (int i=0; i<n; i++) {
			System.out.printf("%d\n", maiorlinha[i]);
	    }

		sc.close();
	}
}
